core-bundle/contao/library/Contao/Model.php

```diff
@@ -841,7 +841,7 @@ public function onRegister(Registry $registry)
 		{
 			$varAliasValue = $this->{$strColumn};
 
-			if (!$registry->isRegisteredAlias($this, $strColumn, $varAliasValue))
+			if (null !== $varAliasValue && !$registry->isRegisteredAlias($this, $strColumn, $varAliasValue))
```

Comment:
```suggestion
			if (!$registry->isRegisteredAlias($this, $strColumn, $varAliasValue ?? ''))
```

Why not this? Just out of curiosity 😄

Reply:
Because an empty string is a valid alias value (e.g. an empty DNS of a root page), but we wouldn't want to fetch this .